sql/item_mods.sql

```diff
@@ -2921,8 +2921,9 @@ INSERT INTO `item_mods` VALUES (10972,10,5);
 INSERT INTO `item_mods` VALUES (10973,1,8);
 INSERT INTO `item_mods` VALUES (10973,30,5);
 INSERT INTO `item_mods` VALUES (10973,233,1);
-INSERT INTO `item_mods` VALUES (10974,1,9);
-INSERT INTO `item_mods` VALUES (10974,31,5);
+INSERT INTO `item_mods` VALUES (10974,1,9); -- Mujin Mantle - DEF +9
+INSERT INTO `item_mods` VALUES (10974,31,5); -- Magic Evasion +5
+-- INSERT INTO `item_mods` VALUES (10974,?,-1); -- Todo - Ninjutsu recast delay -1
```

Comment:
could you open issues for these missing parts? that way they don't get forgotten and lost.

Reply:
Alright, opened one for each of them(2).